### ISSUE TITLE
create-cms-database issues

### ISSUE THREAD

**Issue by alexjwatts83** [episerver/netcore-preview#25]
Hi All,

### TLDR
1. Why does running `create-cms-database` create a user which doesn't have permissions to modify the db?
2. Why is the db created empty?

### `create-cms-database` Errors
This could potentially go down as user error or just my lack of knowledge but I am having trouble getting the alloy cms template to work. I used the instructions to create a alloy site project successfully but after running the command to create the db, I am having issues. 

`dotnet-episerver create-cms-database TestCMSNetCore.csproj -S . -E -v`

Gives me the following output.

![image](https://user-images.githubusercontent.com/20120213/110730947-85a68780-8275-11eb-872a-112c7a970e3a.png)

When I first pressed F5 to debug the site, it came up with the error.

`SqlException: The EXECUTE permission was denied on the object 'sp_DatabaseVersion', database 'master', schema 'dbo'.`

For simplicity sake I made the new user a sysadmin, then pressed F5 again to debug but then got the following error:

`InvalidOperationException: Failed to execute script because validation failed: 'Not an EPiServer database'`

In MSSQL Managment Studio I checked the db and it is empty.

![image](https://user-images.githubusercontent.com/20120213/110731135-db7b2f80-8275-11eb-818b-d49d64b0fb99.png)

I currently have MSSQL Developer edition 2019 installed.

**Am I doing something wrong, why does running create-cms-database create a user which doesn't have permissions to modify the db and why is the db created empty. I was expecting some tables, views, stored procedures, etc.**

_FYI my timezone is, Australian Eastern Standard Time (AEST)_
